### ISSUE TITLE
Cyberpunk mode doesn't work

### ISSUE THREAD

**Issue by spiel0meister** [DexterHuang/CyberCodeOnline#2676]
When I activate Cyberpunk Mode (from Normal mode), click save and the page refreshes, nothing changes.

**Comment by P0L3-5T4R** [comment]
"Cyberpunk Mode" _is_ the "normal mode" now. There aren't any other modes, which is the option in the settings doesn't do anything.

**Comment by howar31** [comment]
So the "VSCode mode" is not available anymore?

**Comment by P0L3-5T4R** [comment]
> So the "VSCode mode" is not available anymore?

That's correct. It was removed quite a while ago now. My understanding is that Dex was finding it far too difficult to maintain as the game grew in scope.
